apps/system/lockscreen/raptor/lockscreen.esp.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = function(filepath, outpath, requiredir) {
+  var opts = {};
+  if (outpath) {
+    var fs = require('fs');
```

Comment:
If you don't need that module in some cases, to require it is only wasting of time. I can take a look to see if the condition is necessary: it's related to the internal logic of Debuguy + Espect in my recollection.